api_gateway/example/cognito.tf

```
@@ -0,0 +1,66 @@
+resource "aws_cognito_user_pool" "pool" {
+  name = "${local.namespace}"
+
+  admin_create_user_config = {
+    allow_admin_create_user_only = true
+  }
+
+  password_policy = {
+    minimum_length = 8
+  }
+}
+
+resource "aws_cognito_user_pool_domain" "domain" {
+  domain       = "apigwexample"
```

Did you meant to hard-code this string? For the pool it’s a variable, `${local.namespace}`. Either this is an example, or it isn’t.

```suggestion
  domain       = "${local.namespace}"
```

---

As I remember, this value needs to not have hyphens in it - which is why I don't reuse it.